smartmon.sh

```diff
@@ -12,7 +12,7 @@
 
 parse_smartctl_attributes_awk="$(
   cat <<'SMARTCTLAWK'
-$1 ~ /^ *[0-9]+$/ && $2 ~ /^[a-zA-Z0-9_-]+$/ {
+$1 ~ /^ *[0-9]+$/ && $2 ~ /^[a-zA-Z0-9_-]+.*/ {
```

Comment:
Let's not open this up to random character matching.

```suggestion
$1 ~ /^ *[0-9]+$/ && $2 ~ /^[a-zA-Z0-9_-]+%?$/ {
```